frontend/src/helpers/chemical-formatters.js

```diff
@@ -17,4 +17,15 @@ export default function chemicalFormula(formula, charge) {
     form = form.replace(/([-+])$/, '<sup>$1</sup>');
   }
   return form;
-}
+};
+
+const chemicalEquation = formula => {
+  let equation = formula.replace('<=>', '⇔');
+  equation = equation.replace('=>', '⇒');
+  equation = equation.replaceAll(/(\S)\+/g, '$1<sup>+</sup>');
```

Comment:
Fe2+ in the equation for `/gotenzymes/reaction/R00078` is also incorrectly formatted. Maybe we can ignore formatted of the charge as well.

Reply:
Comparison of GotEnzymes page vs. GEM Browser page below. I assume there is not much we can do about the order, `Oxygen` vs. `O2`, and the missing compartments. Is the only problem here that some numbers are not wrapped in `sup` tags?

<img width="315" alt="image" src="https://user-images.githubusercontent.com/423498/191241637-2f84127e-6c24-4dec-94f3-601320bc67e0.png">
<img width="397" alt="image" src="https://user-images.githubusercontent.com/423498/191241576-878fdefe-7cb0-4b7f-a594-31a93f87cae1.png">

Reply:
ec75ed5 improves this to 
![2022-09-20-131447_505x64_scrot](https://user-images.githubusercontent.com/1029190/191244638-afd8e50c-ce77-4938-94c8-ba2e81f792f4.png)

Reply:
This commit solves this problem but may bring in some new problems. But I guess the new problems will be in the same category as mentioned in the comment https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1139#discussion_r975111855. I can approve it and then we leave the problem in the new issue. Mostly some edge cases.